docs/topics/running/services/rate-limit-service.md

```diff
@@ -91,15 +91,15 @@ limiting service.
 >
 > ```diff
 >  import (
->  	envoy_ratelimit_v1 "github.com/datawire/ambassador/pkg/api/envoy/service/ratelimit/v1"
+>  	envoy_ratelimit_v1 "github.com/datawire/ambassador/pkg/api/pb/lyft/ratelimit"
 > +	envoy_ratelimit_v2 "github.com/datawire/ambassador/pkg/api/envoy/service/ratelimit/v2"
 >  )
 > ...
 >  	envoy_ratelimit_v1.RegisterRateLimitServiceServer(myGRPCServer, myRateLimitImplementation)
 > +	envoy_ratelimit_v2.RegisterRateLimitServiceServer(myGRPCServer, myRateLimitImplementation)
 > ```
 
-[`v1/rls.proto`]: https://github.com/datawire/ambassador/tree/master/api/envoy/service/ratelimit/v1/rls.proto
+[`v1/rls.proto`]: https://github.com/datawire/ambassador/tree/master/api/pb/lyft/ratelimit/rls.proto
```

Comment:
I would suggest changing `master` here to `release/v1.5`, instead of changing the latter part of the path.

Reply:
OK, I've added a new variable as suggested and made use of it for "Edit this page on GitHub" links as well. Previews for 1.5 are still hosed but you can see the preview for similar changes at https://github.com/datawire/ambassador/pull/2953.